lib/src/main/java/me/ibrahimsn/lib/BottomBarParser.kt

```diff
@@ -9,22 +9,17 @@ import org.xmlpull.v1.XmlPullParserException
 class BottomBarParser(private val context: Context, res: Int) {
 
     private val parser: XmlResourceParser = context.resources.getXml(res)
-    private val items: ArrayList<BottomBarItem> = arrayListOf()
 
-    fun parse(): ArrayList<BottomBarItem> {
-        try {
-            var eventType: Int?
-            do {
-                eventType = parser.next()
-                if (eventType == XmlResourceParser.START_TAG && "item" == parser.name) {
-                    items.add(getTabConfig(parser))
-                }
-            } while (eventType != XmlResourceParser.END_DOCUMENT)
-        } catch (e: XmlPullParserException) {
-            e.printStackTrace()
-            throw Exception()
```

Comment:
Don't do this. If you catch an exception, do something useful with it.
Furthermore ist does never make sense to throw a plain `Exception`.